File: nivel-01/ex-04/reajuste_salario.c
#include <stdio.h>
#include <assert.h>

float reajustar_salario(float salario, float reajuste) {
    return salario * reajuste;
}

int main (){
    
    assert(150.0f == reajustar_salario(1000, 0.15));

    return 0;
}

File: nivel-01/ex-04/reajuste_salario.js
var assert = require('assert');

var reajustar_salario = function(salario, reajuste) {
    return salario * reajuste;
}

try {
    assert.equal(150, reajustar_salario(1000, 0.15));
} catch(e) {
    console.log(e);
}